monitoring/mongodb/alerts.test.yaml

```diff
@@ -333,10 +333,6 @@ tests:
   - name: MongoDbRSNotSynced
     interval: 1m
     input_series:
-      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-0", member_state="SECONDARY"}
-        values: 0x10
-      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-1", member_state="SECONDARY"}
-        values: 0x10
```

Comment:
why remove these?

to test the behavior, we need input series which are consistent with the actual production situation : so need to have multiple series, including all the labels you use to make sure we validate it works in exactly the right cases.

→ we aggregate by `rs_nm`, so we should check this aggregation works
→ this typically happens in 3-node cluster, so we should have the 3 series for mongo0, mongo1 and mong2

in this specific case, the fix you introduce is to add an `instance` : so the series should include it as well, possible with extra series to validate we do not unexpectedly "merge" what should be separate

Reply:
Done here : https://github.com/scality/Zenko/pull/2160/commits/5bbb61af3a8c86e18d2f5018a175fe9ac03b315e#diff-94c90083eba829343ee5745d0056d887a02d888847ea33af739a3ef5c3ae92c6R336-R345